README.md

```diff
@@ -107,171 +114,173 @@ Server running... press ctrl-c to stop.
-5. If there are still problems, please open an [Issue](https://help.github.com/articles/creating-an-issue/) on this repository.
+    ```cmd
+    choco install ruby
+    ```
 
-## Build using Vagrant
+1.  Verify the environment variables were added properly:
 
-You can deploy the devdocs site locally using [this Vagrant project](https://github.com/magento-devdocs/vagrant-for-magento-devdocs).
+    -  In the Windows UI, open search and type `path`.
+    -  In the Windows CMD console, type `echo %path%`.
 
-***
+>  **NOTE**  
+>  If you encounter problems with Ruby, or the `gem` command is not recognized, you can install the `rubyinstaller-devkit.exe` development kit located in the `C:\ProgramData\chocolatey\bin` folder.
 
-If you have questions, open an issue and ask us. We're looking forward to hearing from you!
+### Install Git for Windows
 
-- [Follow @MagentoDevDocs](https://twitter.com/MagentoDevDocs)
-- [E-mail us](mailto:DL-Magento-Doc-Feedback@magento.com)
-- [Visit our documentation site](https://devdocs.magento.com), built using [GitHub pages](https://pages.github.com/).
+Use [Git for Windows](https://gitforwindows.org/) to prevent interference with the existing Windows environment and to have _Git Bash_ and _Git GUI_ launch commands available on the shortcut menu.
 
-## Build on Windows
+Open the Git Setup file downloaded from the Git for Windows site and use the following settings during the installation wizard:
 
-Windows does not provide the out-of-the-box Ruby, so we recommend to use a virtual environment with Docker or Vagrant described above.
-But if none of the ways work for you, consider using the following tools.
+-  select **Use Git from Git Bash only**
+-  select **Checkout as-is, commit Unix-style line endings**
+-  select your preferred editor (can use Nano, Notepad++, or VIM)
+-  select **Enable symbolic links**
 
-- [Git for Windows](https://git-scm.com/download)
-- [Chocolatey](https://chocolatey.org)
+Although you can install Git using Chocolatey, we chose to install _Git for Windows_ independently for more control of the installation settings.
 
-### Install Chocolatey
+#### Set up SSH key
 
-1. Open the **Command Prompt** using **Run as Administrator** in the shortcut menu.
+1.  Open Git Bash. The **Git Bash** executable is on the shortcut menu.
 
-2. [Install Chocolatey](https://chocolatey.org/install).
+1.  Create a working directory for your Git repositories and change to the new directory.
 
-3. Verify Chocolatey was added to the environment variables. In the Windows CMD console, type `echo %path%`.
-   You should see `C:\ProgramData\chocolatey\bin` in the path.
+    ```bash
+    mkdir <directory-name>
+    ```
 
-4. Close and reopen the command prompt before using `choco` commands.
+1.  Follow the [Generating a new SSH](https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/) instructions.
 
-### Install Ruby package
+### Clone and build the DevDocs repository
```

Comment:
This scenario is exactly the same as for building devdocs on Unix.
It makes sense to just add a link to the section above to not duplicate the content.

Reply:
No. That makes them hop around. I don't want to break the flow.

Reply:
It makes the scenario overloaded with information and requires more maintenance in case of any changes. Also it makes a user to think that the Windows scenario is different and much more complex.

Reply:
If there's no difference between the instructions, then I agree that we shouldn't duplicate content.

Reply:
I found a way that did not feel like we were forcing them to hop around.